GUI & renewables - time series management

guilpier-code opened the pull request:
This PR concerns the **GUI > Simulation > Time-Series Management** grid.
A switch in **Configure > Advanced parameters > Other Preferences** must be added.
This switch allows user to choose if she wants to use old renewables (wind and Solar), or the renewable clusters.
In the second case,  **GUI > Simulation > Time-Series Management** grid is changed : Wind and Solar columns desappear and a
**Renewable** column appears.

guilpier-code committed:
UI time-series management : adding the switch between old and new renewables

guilpier-code committed:
UI time-series management : moving ts management grid creation

This is originally a test to see if ts management grid creation could be moved from constructor to another method
where we are sure that the study is loaded.
It will be useful for next development, so we keep it as a commit.

guilpier-code committed:
UI time-series management : renaming TS management renderer files with more proper names

guilpier-code committed:
UI time-series management : renaming class for the TS management renderer

Old name of class "SimulationTSManagement" is changed into "TSmanagement", shorter.

guilpier-code committed:
UI time-series management : correcting a previous commit (files renaming)

flomnes reviewed (commented):
TODO :
- Take the parameter into account in **SIM**, when filling structures with TS values.
- Disable TS generation for wind and solar
- Disable scenario building for wind and solar
- Disable intra/intermodal TS number picking (see timeseries-numbers.cpp)

guilpier-code committed:
UI time-series management : time series management conditional display

Displaying a different time series management grid depending the value of advanced parameter : renewable generation modeling
- if it is set to "aggregated", the grid is the usual one
- if it is set to "clusters", the Wind and Solar usual columns are replaced with a "Renewable" column (for clusters)
